### PR TITLE
feat(Core/Common): delete old Tokenizer

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Changes Proposed:
- Partical cherrypick https://github.com/TrinityCore/TrinityCore/commit/534a2388b7c662c8796aabb1ec8cb424879799b6
- Fix startup db errors
```
Loading Creature template addons...
Creature (Entry: 30680) has wrong spell '57742,' defined in `auras` field in `creature_template_addon`.
Creature (Entry: 30999) has wrong spell '57742,' defined in `auras` field in `creature_template_addon`.
```

## Issues Addressed:
-

## Tests Performed:
- Tested in game

# How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1641914952623130600.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1641914952623130600');
+
+UPDATE `creature_template_addon` SET auras = REPLACE(auras, ',', ' ');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1641914952623130600.sql:3]: Backticks

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1641914952623130600.sql (hunk 1 above)
Comment (Winfidonarleyan) [reply data/sql/updates/pending_db_world/rev_1641914952623130600.sql:3]: for auras?

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1641914952623130600.sql:3]: Yes

Comment (Winfidonarleyan) [inline data/sql/updates/pending_db_world/rev_1641914952623130600.sql:3]:
```suggestion
UPDATE `creature_template_addon` SET `auras` = REPLACE(auras, ',', ' ');
```

File: data/sql/updates/pending_db_world/rev_1641914952623130600.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1641914952623130600');
+
+UPDATE `creature_template_addon` SET `auras` = REPLACE(auras, ',', ' ');
```
Comment (Winfidonarleyan) [inline data/sql/updates/pending_db_world/rev_1641914952623130600.sql:3]: @Kitzunu can check? :D

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**